docs/variants/pc_engines/initial-deployment.md

```diff
@@ -0,0 +1,42 @@
+# Initial deployment
+
+Initial flashing of Dasharo firmware can be done from Linux using flashrom with
+the internal programmer.
+
+## Deploy using Dasharo Tools Suite
+
+For simplicity we recommend using
+[Dasharo Tools Suite](../../dasharo-tools-suite/overview.md) to
+omit all compilation steps and deploy the Dasharo firmware seamlessly.
+
+1. Boot Dasharo Tools Suite.
+2. Perform Dasharo installation.
+
+After the successful operation, DTS will reboot the platform.
+
+This concludes Dasharo deployment process using DTS.
+
+## Build flashrom
+
+Please follow generic guide for [Dasharo flashrom fork](../../osf-trivia-list/deployment.md#how-to-install-dasharo-flashrom-fork).
```

Comment:
I doubt it is necessary for PC Engines, but I may be missing something.

Reply:
It is not necessary to use fork indeed.